src/Network/Wai/SAML2/Assertion.hs

```diff
@@ -82,7 +82,9 @@ instance FromXML SubjectConfirmation where
 -- | The subject of the assertion.
 data Subject = Subject {
     -- | The list of subject confirmation elements, if any.
-    subjectConfirmations :: ![SubjectConfirmation]
+    subjectConfirmations :: ![SubjectConfirmation],
+    -- | The name identifier of subject.
```

Comment:
Perhaps it's better to mention the term "NameID" instead of "name identifier".

Reply:
resolved with https://github.com/mbg/wai-saml2/pull/13/commits/05139ac9659523086a2f32963deebc001cbf57fd